medical_prescription/tests/test_medical_prescription_order_line.py

```diff
@@ -43,3 +44,18 @@ def test_name_search_none(self):
             len(recs),
             'Length of recs (%d) not equal to 3' % (len(recs)),
         )
+
+    def test_prescription_order_id_required(self):
```

Comment:
There's not as long as we have demo data - that will trigger the creations and yield any errors. We can assume something simple like a `required=True` won't be changed erroneously

You are technically testing something and this is technically valuable - hence me not saying remove. Just a slippery slope when we start testing whether model attributes are required.